modules/bezug_rct2/status.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Debughilfe
+if [ -n "$bezug1_ip" ]; then
+  opt=""
+else
+  echo "$0 Debughilfe bezug1_ip parameter not supplied use 192.168.208.63"
```

Comment:
Da außer Dir niemand mit diesem Else-Zweig etwas anfangen kann, sollte er auch nicht im Code bleiben oder zumindest auskommentiert werden.
Vielleicht ist es auch besser, die loadconfig.sh in diesem Fall aufzurufen, wie es in der info.sh gemacht wird?